ibridges/irodsconnector/data_operations.py

```diff
@@ -61,13 +59,13 @@ def obj_replicas(obj: irods.data_object.iRODSDataObject) -> list(tuple(int, str,
         Returns
         -------
         list(tuple(int, str, str. int, str))
-            List with tuple where each tuple contains replica index/number, resource name on which 
-            the replica is stored about one replica, replica checksum, replica size, 
+            List with tuple where each tuple contains replica index/number, resource name on which
+            the replica is stored about one replica, replica checksum, replica size,
             replica status of the replica
         """
         replicas = []
         for r in obj.replicas:
-            replicas.append((r.number, r.resource_name, r.checksum))
+            replicas.append((r.number, r.resource_name, r.checksum, r.size, r.status))
```

Comment:
Is there a way to check for the user what the value of `status` means (I just get a `1`)?

Reply:
It is hard to retrieve that from the irods server software. This is how far we got:
OBJ_STATUS_SYMBOL = {
    '0': 'X', # replica checksum check failure
    '1': '&', # replica checksums in sync with other replicas
}
I also found some higher codes. For interpretation I would say, if the status is 1 all is fine, everything else is fishy.

Reply:
This is from the `ils` help:
```
   0. X: stale        - The replica is no longer known to be good. Usually [...]
   1. &: good         - The replica is good. Usually the latest replica to be [...]
   2. ?: intermediate - The replica is actively being written to. The state of [...]
   3. ?: write-locked - One of this replica's sibling replicas is actively [...]
```
Maybe we can add the labels (`stale', `good` etc) for clarity (and users don't mistake 0 for the "best" status).